packages/grafana-ui/src/components/TimePicker/TimeZonePicker.tsx

```diff
@@ -6,11 +6,12 @@ import { FormInputSize } from '../Forms/types';
 interface Props {
   value: string;
   size?: FormInputSize;
+  includeDefault?: boolean;
```

Comment:
What does this mean? Include local ?

Reply:
@mckn -- this was started aiming to support using the non-standard timezones we use:

![image](https://user-images.githubusercontent.com/705951/80401196-c7525d80-8870-11ea-94f8-13da3bfd61d8.png)